src/camera.rs

```diff
@@ -603,6 +642,15 @@ impl Camera {
 
         /// Recent Frame mode.
         mut recent_frame: XI_SWITCH::Type;
+
+        /// Setting the API debug level allows to select amount of messages stored to debug output.
+        mut debug_level: XI_DEBUG_LEVEL::Type;
```

Comment:
```suggestion
/// Removed
```
I would not add this parameter to the camera and here is why:

You can already set the debug level using the static top level xiapi::set_debug_level function. The debug level in xiAPI is always global, so it does not make sense to have it on the camera. The static function has the additional advantage that you can use it before even openining a camera, so you can get debug output from the beginning.

Reply:
Wow, somehow I missed `set_debug_level`! Updated